BeeTracker/app/src/main/java/com/blueprint/foe/beetracker/ConfirmEmailFragment.java

```diff
@@ -46,7 +56,67 @@ public void onClick(View view) {
             }
         });
 
+        resendCallback = new Callback<BeeTrackerCaller.ResendEmailResponse>() {
+            @Override
+            public void onResponse(Call<BeeTrackerCaller.ResendEmailResponse> call, Response<BeeTrackerCaller.ResendEmailResponse> response) {
+                if (response.code() == 401 || response.body() == null) {
+                    Log.e(TAG, "The response from the server is 401 + " + response.message());
+                    showErrorDialog(getString(R.string.error_message_resend_email));
+                    return;
+                }
+
+                if (spinningIconDialog != null) {
+                    spinningIconDialog.dismiss();
+                }
+
+                Toast.makeText(getActivity(), getString(R.string.confirm_email_resent, email), Toast.LENGTH_SHORT).show();
+            }
+
+            @Override
+            public void onFailure(Call<BeeTrackerCaller.ResendEmailResponse> call, Throwable t) {
+                Log.e(TAG, "There was an error with the signup callback + " + t.toString());
```

Comment:
The log statement should probably say "resend email callback" or something to that effect